week1/gyuunindex.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>1주차 실습 : turtlehwan 자기소개</title>
```

Comment:
와 1등으로 PR 해주셨네요
빠른 첫 PR 아주 좋습니다~!

다만 여기 부분이 turtlehwan으로 되어 있네요! 
Hwanggyuun 등 자신을 나타낼 수 있는 이름으로 바꾸면 더욱 좋을 거 같아요
```suggestion
    <title>1주차 실습 : Hwanggyuun 자기소개</title>
```